production/exit-node-1/application.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: exit-node-1
+  namespace: production
+spec:
+  project: production
+  source:
+    repoURL: https://github.com/Rpc-h/helm-charts
+    targetRevision: main
+    path: exit-node
+    plugin:
+      env:
+        - name: HELM_VALUES
+          value: |
+            image:
+              repository: europe-west6-docker.pkg.dev/rpch-375921/rpch/exit-node
+              tag: "2969662"
```

Comment:
How is this tag generated? You should, not now, but eventually move to use other pattern that is more meaningfull